test/doiuse.js

```diff
@@ -0,0 +1,18 @@
+var test = require('tape')
+var mock = require('mock-fs')
+var doiuse = require('../lib/doiuse')
+
+test('info with browserslist file', function (t) {
+  mock({
+    'browserslist': 'Safari 8\nIE >= 11'
+  })
+
+  var actual = doiuse({}).info().browsers
+  var expected = [['ie', '11'], ['safari', '8']]
+
+  t.deepEqual(actual, expected)
+
+  mock.restore()
+
+  t.end()
+})
```

Comment:
This test could go into the (admittedly poorly named) file `test/postcss-plugin.js`. (Optionally, that file could be renamed to `test/doiuse.js` first, so that it matches the module it's testing.)

Also, related to the comment about the `while` loop above, could we add a test for the case where neither a `browserslist` file nor a `browsers` option is provided?